cfgov/v1/models/enforcement_action_page.py

```diff
@@ -0,0 +1,305 @@
+    (
+        'CFPA Unfair',
+        'Consumer Financial Protection Act - Unfair Acts or Practices'
+    ),
+    (
+        'CFPA Abusive',
+        'Consumer Financial Protection Act - Abusive Acts or Practices'
+    ),
+    ('CFPA', 'Consumer Financial Protection Act - Other'),
+    ('AMTPA', 'Alternative Mortgage Transaction Parity Act/Regulation D'),
+    ('CLA', 'Consumer Leasing Act/Regulation M'),
+    ('Credit Practice Rules', 'Credit Practices Rule'),
+    ('EFTA/Regulation E', 'Electronic Fund Transfer Act/Regulation E'),
+    ('ECOA/Regulation B', 'Equal Credit Opportunity Act/Regulation B'),
+    ('FCBA', 'Fair Credit Billing Act'),
+    ('FCRA/Regulation V', 'Fair Credit Reporting Act/Regulation V'),
+    ('FDCPA', 'Fair Debt Collection Practices Act/Regulation F'),
+    ('FDIA', 'Federal Deposit Insurance Act/Regulation I'),
+    ('GLBA/Regulation P', 'Gramm-Leach-Bliley Act/Regulation P'),
+    ('HMDA', 'Home Mortgage Disclosure Act/Regulation C'),
+    ('HOEPA', 'Home Ownership and Equity Protection Act'),
+    ('HOPA', 'Home Owners Protection Act'),
+    (
+        'ILSFDA',
+        'Interstate Land Sales Full Disclosure Act/Regulation J, K, and L'
+    ),
+    ('Military Lending Act', 'Military Lending Act'),
+    (
+        'Regulation N (MAP Rule)',
+        'Mortgage Acts and Practices – Advertising Final Rule (Regulation N)'
+    ),
+    (
+        'Regulation O (MARS Rule)',
+        'Mortgage Assistance Relief Services Rule (Regulation O)'
+    ),
+    ('MRAPLA', 'Mortgage Reform and Anti-Predatory Lending Act'),
+    ('RESPA', 'Real Estate Settlement Procedures Act/Regulation X'),
+    ('SMLA', 'S.A.F.E. Mortgage Licensing Act/Regulation H'),
+    ('Telemarketing Sales Rule (TSR)', 'Telemarketing Sales Rule'),
+    ('TILA/Regulation Z', 'Truth in Lending Act/Regulation Z'),
+    ('TISA/Regulation DD', 'Truth in Savings Act/Regulation DD')
+]
+
+
+def decimal_field():
+    return models.DecimalField(
+        decimal_places=2,
+        max_digits=13,
+        default=0
+    )
+
+
+class EnforcementActionDisposition(models.Model):
+    final_disposition = models.CharField(max_length=150, blank=True)
+    final_disposition_type = models.CharField(
+        max_length=15,
+        choices=[('Final Order', 'Final Order'), ('Dismissal', 'Dismissal')],
+        blank=True
+    )
+    final_order_date = models.DateField(null=True, blank=True)
+    dismissal_date = models.DateField(null=True, blank=True)
+    final_order_consumer_redress = decimal_field()
+    final_order_consumer_redress_suspended = decimal_field()
+    final_order_other_consumer_relief = decimal_field()
+    final_order_other_consumer_relief_suspended = decimal_field()
+    final_order_disgorgement = decimal_field()
+    final_order_disgorgement_suspended = decimal_field()
+    final_order_civil_money_penalty = decimal_field()
+    final_order_civil_money_penalty_suspended = decimal_field()
+    estimated_consumers_entitled_to_relief = models.CharField(
```

Comment:
This field is causing issues when I try to edit some pages that have been updated. It seems like some lines of the csv are blank for this field, so this field gets an empty string, which isn't a valid value. Then, when I try to edit that page in the Wagtail editor, the page won't save because of the validation error.

Maybe this field should have `blank=True`, or the script should check for a blank value?

Reply:
hmmm that's odd considering there's a default value set. You have the latest code/migrations and have run them?Incidentally the default value should be changed to `blank=True` anyway since this is a CharField now.